### PR TITLE
[library] Add highlight component

### PR DESCRIPTION
**Pull request by juliawu**
Adds the ability to render a highlight tile as a web component.

This PR:
* Adds a highlight component that wraps the highlight tile 
* Adds the ability to specify a specific date in the highlight tile
* Adds a "part" to the highlight tile containing div to allow for web component styling via CSS
* Adds documentation for the new highlight component
* Also fixes an error in the map web component documentation (date attribute should be under optional, not required)

Now the syntax
```
<datacommons-highlight
  description="Population of the United States"
  place="country/USA"
  variable="Count_Person"
  date="2020"
></datacommons-highlight>
```
Produces:

Screenshot 2023-08-04 at 4 45 53 PM

### CODE REVIEW

File: packages/web-components/docs/components/highlight.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Data Commons Gauge Chart Web Component
```
Comment (dwnoble) [inline packages/web-components/docs/components/highlight.md:1]: s/Gauge/Highlight/

File: packages/web-components/docs/components/highlight.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+<datacommons-highlight
+  title="Percentage of US Population that are Internet Users"
+  place="country/USA"
+  variable="Count_Person_IsInternetUser_PerCapita"
+  min="0"
+  max="100"
+></datacommons-highlight>
+```
+
+<img src="../assets/gauge.png" width="620"/>
```
Comment (dwnoble) [inline packages/web-components/docs/components/highlight.md:18]: s/gauge/highlight/

**Review by dwnoble — APPROVED** [review]
Thanks Julia! I think @pradh added something to the highlight tile the other day to show the unit-can you verify this works with the web component?

File: packages/web-components/docs/components/highlight.md (hunk 1 above)
Comment (juliawu) [reply packages/web-components/docs/components/highlight.md]: Thanks for the catch!

File: packages/web-components/docs/components/highlight.md (hunk 2 above)
Comment (juliawu) [reply packages/web-components/docs/components/highlight.md]: Thanks for the catch! Added a new image and linked to that instead.

**Comment by juliawu** [comment]
> Thanks Julia! I think @pradh added something to the highlight tile the other day to show the unit-can you verify this works with the web component?

The unit will show in the tile when a unit is passed into the statvar spec. Added an optional attribute to set a "unit" string to display to the web component.

**Pull request merged**